.storybook/components/DesignTokens/Legacy/TokenTable.tsx

```diff
@@ -33,4 +33,6 @@ const TokenTable = ({ tokens, renderExample }: TokenTableProps) => {
   );
 };
 
+// Stories exist as mdx extension
```

Comment:
The eslint rule itself can be updated for the mdx extension but we can reconsider later since mdx processing requires `@storybook/addon-docs` and loses typescript typing and isn't being used in consumers

Reply:
I don't really understand this comment 🙃 

What is the "mdx extension" you are referring to? How is this file being used in storybook right now?

Reply:
Sorry, to clarify, this component is being represented in `LegacyColor.stories.mdx`. Our lint rule will not catch this since it has a different file name and a file extension of `.mdx` and not `.ts` or `.tsx`. The lint rule itself probably doesn't need updating to support the `.mdx` file extension since we want type safe stories and `.stories.mdx` is only used in  `LegacyColor.stories.mdx` and `LegacyTypography.stories.mdx` of EDS.